cabot/cabotapp/jenkins.py

```diff
@@ -14,23 +15,27 @@ def _get_jenkins_client():
 
 def get_job_status(jobname):
     ret = {
-        'active': True,
-        'succeeded': False,
+        'active': None,
+        'succeeded': None,
```

Comment:
Yeah, I’m not really happy with that... The thing is that `run` kind of expects some keys to be there or crashes horribly − which just falls through a success. Pending some actual revamp of that method, setting these keys was just easier, and since it felt wrong to give actual values when none would make sense, I  went for this.